### ISSUE TITLE
Document kube:admin VS htpasswd_provider when logging in the web console

### ISSUE THREAD

**Issue by gasgithub** [code-ready/crc#980]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V 
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
# Put `crc version` output here

PS C:\crc-1.4.0> crc version
crc version: 1.4.0+d5bb3a3
OpenShift version: 4.2.13 (embedded in binary)
```

  
## CRC status
```bash
# Put `crc status` output here
PS C:\crc-1.4.0> crc status
CRC VM:          Running
OpenShift:       Running (v4.2.13)
Disk Usage:      11.66GB of 32.2GB (Inside the CRC VM)
Cache Usage:     13.9GB
Cache Directory: C:\Users\userName\.crc\cache
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
PS C:\crc-1.4.0> systeminfo
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.18362 N/A Build 18362
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~2592 Mhz
Windows Directory:         C:\Windows
System Locale:             en-us;English (United States)
Time Zone:                 (UTC+01:00) Sarajevo, Skopje, Warsaw, Zagreb
Total Physical Memory:     130 879 MB
Available Physical Memory: 99 295 MB
Virtual Memory: Max Size:  150 335 MB
Virtual Memory: Available: 115 985 MB
Virtual Memory: In Use:    34 350 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\abc
Network Card(s):           5 NIC(s) Installed.
                           [01]: Intel(R) Ethernet Connection (7) I219-V
                                 Connection Name: Ethernet
                                 Status:          Media disconnected
                           [02]: Intel(R) Wi-Fi 6 AX200 160MHz
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.0.1
                                 IP address(es)
                                 [01]: 192.168.0.241
                                 [02]: fe80::19bf:8e97:6959:274f
                                 [03]: 2a02:a311:a33f:b80:bd8a:db4b:bc4f:d6a8
                                 [04]: 2a02:a311:a33f:b80:19bf:8e97:6959:274f
                           [03]: Cisco AnyConnect Secure Mobility Client Virtual Miniport Adapter for Windows x64
                                 Connection Name: Ethernet 2
                                 Status:          Hardware not present
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.17.226.193
                                 [02]: fe80::e012:3095:6ffa:572f
                           [05]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (DockerNAT)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 10.0.75.1
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.


```

### Steps to reproduce

  1. crc is successfully started and I get the following messages at the end:

INFO Starting OpenShift cluster ... [waiting 3m]
INFO
INFO To access the cluster, first set up your environment by following 'crc oc-env' instructions
INFO Then you can access it by running 'oc login -u developer -p developer https://api.crc.testing:6443'
INFO To login as an admin, run 'oc login -u kubeadmin -p abc https://api.crc.testing:6443'
INFO
INFO You can now run 'crc console' and use these credentials to access the OpenShift web console

  2.  I can successfully log in from oc on developer:

PS C:\crc-1.4.0> oc login -u developer -p developer https://api.crc.testing:6443
Login successful.

You don't have any projects. You can try to create a new project, by running

  3. I invoke `crc console` to open web console
  4. After typing username: developer with password: developer I got `invalid login or password` message. However I can successfully login to console using kubeadmin account

### Expected
Login on developer account works from web browser.

### Actual
I got `invalid login or password` message. 

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by praveenkumar** [comment]
@gasgithub Make sure you use the `htpasswd_provider` link and not the `kube:admin` one.

![Screenshot from 2020-01-29 17-08-17](https://user-images.githubusercontent.com/410007/73353811-3d6b1680-42ba-11ea-8045-c808ea1cbda5.png)

**Comment by gasgithub** [comment]
Thank you for prompt response. Yes, it works correctly. I somehow missed that, sorry for bothering.

**Comment by praveenkumar** [comment]
I am reopening this issue to add documentation around it so users shouldn't be confuse about it.

**Comment by robin-owen** [comment]
Related to #679.

**Comment by cfergeau** [comment]
Indeed, let's close this one as a duplicate of #679

**Comment by cfergeau** [comment]
Duplicate of #679
